packages/application-extension/src/index.tsx

```diff
@@ -42,6 +46,14 @@ namespace CommandIDs {
   export const activatePreviousTab: string =
     'application:activate-previous-tab';
 
+  export const close = 'application:close';
+
+  export const closeAllFiles = 'docmanager:close-all-files';
```

Comment:
Do we need this at all? It doesn't looks like it actually does what it says (instead it seems duplicative of the application close all command). We could either (1) remove it, or (2) make it do what the label says.

Reply:
I think (1) we don't